api/model.go

```diff
@@ -542,6 +543,16 @@ func toSettingCollection(settings []*longhorn.Setting) *client.GenericCollection
 	return &client.GenericCollection{Data: data, Collection: client.Collection{ResourceType: "setting"}}
 }
 
+func getReplicaName(address string, vrs []*longhorn.Replica) string {
+	for _, r := range vrs {
+		if strings.Contains(address, r.Status.IP) {
+			return r.Name
+		}
+	}
+	logrus.Errorf("BUG: invalid replica address[%v] received from engine restoreStatus", address)
+	return ""
```

Comment:
In the case of error, return the original address instead. So at least we know what's the address.

Reply:
I'm thinking of removing this log as it's technically not a BUG. If a volume is in `detached` state, it wouldn't have any replica at that moment. So I believe, it would be right to show it as `empty` instead of the IP where it was previously healthy at. What do you think?